src/components/Cards/Cards.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Data from "./cardData";
+
+import Card1 from "./Card1";
+
+const lemy = (item) => {
```

Comment:
The COmponent name is lowercase, shouldn't be. Lemy is not a descriptive name, I should have an idea of what your component does by looking at the name.

I commend the creative use of reusable component here.